t/t7900-maintenance.sh

```diff
@@ -1012,4 +1012,129 @@ test_expect_success 'repacking loose objects is quiet' '
 	)
 '
 
+test_expect_success 'vfs-cache-move task with no shared cache no op' '
+	test_when_finished "rm -rf repo" &&
+	git init repo &&
+	(
+		cd repo &&
+
+		test_commit something &&
+		git config set maintenance.gc.enabled false &&
+		git config set maintenance.vfs-cache-move.enabled true &&
+		git config set maintenance.vfs-cache-move.auto 1 &&
+
+		printf "blob\ndata <<END\n%s\nEND\n\n" 1 2 3 4 5 | \
+		git -c fastimport.unpackLimit=0 fast-import &&
+		find .git/objects/pack \
+			-type f \
+			\( -name "*.pack" \
+			-o -name "*.idx" \
+			-o -name "*.keep" \
+			-o -name "*.rev" \) >files.txt &&
```

Comment:
This seems to be repeated code. Could we please move it into a shell function that is then used in those four new test cases?

Reply:
Done